static/js/containers/SettingsPage.js

```diff
@@ -3,15 +3,16 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Loader from '../components/Loader';
+import R from 'ramda';
 
 import { startProfileEdit } from '../actions/profile';
 import { FETCH_PROCESSING } from '../actions/index';
-import ProfileFormContainer from './ProfileFormContainer';
+import { profileFormContainer, mapStateToProfileProps } from './ProfileFormContainer';
 import PrivacyForm from '../components/PrivacyForm';
 import ProfileProgressControls from '../components/ProfileProgressControls';
 import { privacyValidation } from '../lib/validation/profile';
 
-class SettingsPage extends ProfileFormContainer {
+class SettingsPage extends React.Component {
```

Comment:
Can you add a props declaration here since `ProfileFormContainer` is no longer doing it?